lib/dotcom_web/live/system_status.ex

```diff
@@ -6,41 +6,89 @@ defmodule DotcomWeb.Live.SystemStatus do
 
   use DotcomWeb, :live_view
 
+  import DotcomWeb.Components.RoutePills
+  import DotcomWeb.Components.SystemStatus.StatusLabel
+  import DotcomWeb.Components.SystemStatus.Widget
+
   alias Dotcom.SystemStatus
 
   def render(assigns) do
     alerts = SystemStatus.subway_alerts_for_today()
 
     statuses = SystemStatus.subway_status()
 
+    examples =
+      alerts_examples()
+      |> Enum.map(fn %{alerts: alerts} = example ->
+        example |> Map.put(:statuses, SystemStatus.Subway.subway_status(alerts, Timex.now()))
+      end)
+
     assigns =
       assigns
       |> assign(:alerts, alerts)
       |> assign(:statuses, statuses)
+      |> assign(:examples, examples)
+
+    Widget
 
     ~H"""
-    <h1>System Status</h1>
-    <div>
-      <.status :for={status <- @statuses} status={status} />
-    </div>
-    <h1>Alerts</h1>
+    <h1>Live Data</h1>
+    <.system_status_widget routes_with_statuses={@statuses} />
+
+    <h2>Alerts</h2>
     <div class="flex flex-col gap-2">
       <.alert :for={alert <- @alerts} alert={alert} />
     </div>
-    """
-  end
 
-  defp status(assigns) do
-    ~H"""
-    <pre>{inspect @status, pretty: true}</pre>
+    <h1>Examples</h1>
+    <div :for={example <- @examples} class="mb-4">
+      <div class="flex gap-5">
+        <.system_status_widget routes_with_statuses={example.statuses} />
+        <div class="flex flex-col gap-5">
+          <span class="text-lg font-bold">Alerts</span>
+          <.alert :for={alert <- example.alerts} alert={alert} />
+        </div>
+      </div>
+    </div>
+
+    <h1>Misc Components</h1>
```

Comment:
I added these into the preview page for testing convenience, but I'm not attached to keeping them there. Happy to remove this if it's a problem for anyone.